rfc/rfc-000-lightweightRFCprocess.md

```diff
@@ -0,0 +1,76 @@
+# RFC-000: Lightweight RFC Process for riff
+
+**To be Reviewed By:** Nov 15th 2019
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Pull Request URL:** https://github.com/projectriff/riff/pull/1358
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+
+## Problem
+As the scope of the project expands and the number of moving pieces increases, we need a process to gather feedback on the changes proposed to the architecture and interaction of various components in the system.
+
+This document will refer to that process as the “Request For Comments (RFC) process”.
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or a group of authors submit a proposal to the community in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
```

Comment:
...submit a proposal in the form of a RFC PR with a RFC markdown file, _both_ called `RFC-xxx-<some-name>.md`, in order to gather...

Reply:
adopted!